content/en/observability_pipelines/guide/strategies_for_reducing_log_volume.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Strategies for Reducing Log Volume
+disable_toc: false
+further_reading:
+- link: "/observability_pipelines/set_up_pipelines"
+  tag: "documentation"
+  text: "Set up a pipeline"
+- link: "/observability_pipelines/processors/"
+  tag: "documemtation"
+  text: "Observability Pipelines processors"
+---
+
+## Overview
+
+Observability Pipelines allows you to collect and process logs, as well as determine where you want to route them, all before the data leaves your on-premises or cloud environment.
+
+The value of your logs is not all equal. For example, error logs are often more useful than info logs when troubleshooting live operations. Logs from production environments are also more important than logs from non-production environments. Therefore, routing all of your logs to an indexed solution can end up diluting the overall value of your data and cause you to exceed your budget.
```

Comment:
>The value of your logs is not all equal.

This just reads funny to me, I am not an expert on grammar or wordsmithing, so grain of salt, but putting the `all` before `logs` reads better to me, i.e.

> The value of all your logs is not equal

? 

Not sure if that came from my doc, if so then just me talking to myself 😆

Reply:
Thanks for point that out! I was struggling with it before....it read funny to me too. I just updated it to: `Logs are not all equal in value`. What do you think?